source/acquaman/AMGenericStepScanConfiguration.cpp

```diff
@@ -269,6 +269,16 @@ void AMGenericStepScanConfiguration::removeRegionOfInterest(AMRegionOfInterest *
 		}
 }
 
+void AMGenericStepScanConfiguration::setRegionOfInterestBoundingRange(AMRegionOfInterest *region)
+{
```

Comment:
Pedantically speaking, you are correct.  That has been typically omitted in this (and the addRegionOfInterest and removeRegionOfInterest) because the region of interest itself is emitting the signal through `emit signal(this);` and if the region was not valid then that would cause a seg fault.